.travis.yml

```diff
@@ -34,7 +34,7 @@ matrix:
 before_install:
   - echo cakephp version && tail -1 VERSION.txt
 
-  - if [[ $DB == 'mysql' ]]; then mysql -u root -e 'CREATE DATABASE cakephp_test;'; fi
+  - if [[ $DB == 'mysql' ]]; then mysql -u root -e 'CREATE DATABASE cakephp_test CHARACTER SET utf8mb4;'; fi
```

Comment:
This should probably be done for `master` itself as `utf8mb4` is used as default for the Mysql driver https://github.com/cakephp/cakephp/blob/1e0d18199b38f6063ca10d6f7d5aad52ec01ca83/src/Database/Driver/Mysql.php#L66.